common-react/@dbeaver/ui-kit/src/Spin/_base.css

```diff
@@ -0,0 +1,37 @@
+@keyframes stroke {
+  0% {
+    stroke-dasharray: 1px, 100px;
+    stroke-dashoffset: 0;
+  }
+  50% {
+    stroke-dasharray: 45px, 100px;
+    stroke-dashoffset: -10px;
+  }
+  100% {
+    stroke-dasharray: 17px, 100px;
+    stroke-dashoffset: -58px;
+  }
+}
+
+@keyframes spinning {
```

Comment:
can we use animation from tailwindcss? https://tailwindcss.com/docs/animation

Reply:
no, we need to control duration and timing function.